ivy/container/experimental/elementwise.py

```diff
@@ -3225,3 +3225,94 @@ def binarizer(
             map_sequences=map_sequences,
             out=out,
         )
+
+    @staticmethod
+    def static_ldexp(
+        x: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+        i: Union[ivy.Array, ivy.NativeArray, ivy.Container],
```

Comment:
I see you've changed Numpy's arguments to `x1` and `x2` , could you apply those changes here as well ?